code/game/objects/items/weapons/twohanded.dm

```diff
@@ -381,48 +386,47 @@
 			M.GiveTarget(L)
 
 //Putting heads on spears
-/obj/item/organ/external/head/attackby(var/obj/item/W, var/mob/living/user, params)
-	if(istype(W, /obj/item/twohanded/spear))
-		to_chat(user, "<span class='notice'>You stick the head onto the spear and stand it upright on the ground.</span>")
-		var/obj/structure/headspear/HS = new /obj/structure/headspear(user.loc)
-		var/matrix/M = matrix()
-		src.transform = M
-		user.drop_item()
-		src.loc = HS
-		var/image/IM = image(src.icon,src.icon_state)
-		IM.overlays = src.overlays.Copy()
-		HS.overlays += IM
-		qdel(W)
-		return
-	return ..()
-
-/obj/item/twohanded/spear/attackby(var/obj/item/I, var/mob/living/user)
+/obj/item/twohanded/spear/attackby(obj/item/I, mob/living/user)
 	if(istype(I, /obj/item/organ/external/head))
-		to_chat(user, "<span class='notice'>You stick the head onto the spear and stand it upright on the ground.</span>")
-		var/obj/structure/headspear/HS = new /obj/structure/headspear(user.loc)
-		var/matrix/M = matrix()
-		I.transform = M
-		usr.drop_item()

```

Comment:
This code is so bad. Why in the world would you do this. You have a `user` argument, but you use `usr` anyway? What the heck, man.